### PR TITLE
Update .pre-commit-config.yaml

### PR DESCRIPTION
**Pull request by Dreamsorcerer**

### CODE REVIEW

File: aiohttp/web_response.py [hunk 1]
```diff
@@ -176,7 +176,7 @@ def enable_compression(
     ) -> None:
         """Enables response compression encoding."""
         # Backwards compatibility for when force was a bool <0.17.
-        if type(force) == bool:
+        if isinstance(force, bool):
```
Comment (webknjaz) [inline aiohttp/web_response.py:179]: Are integers boolean? I remember there were some corner cases a around this. Or perhaps this was done with performance in mind.. FWIW this is not just a formatting change. It checks for subclasses too. Is this necessary?

File: tests/test_circular_imports.py [hunk 2]
```diff
@@ -30,14 +31,16 @@ def _mark_aiohttp_worker_for_skipping(
     importables: List[str],
 ) -> List[Union[str, "ParameterSet"]]:
     return [
-        pytest.param(
-            importable,
-            marks=pytest.mark.skipif(
-                not hasattr(socket, "AF_UNIX"), reason="It's a UNIX-only module"
-            ),
+        (
```
Comment (webknjaz) [inline tests/test_circular_imports.py:34]: Which linter wanted this? It seems like too much..

File: tests/test_proxy_functional.py [hunk 3]
```diff
@@ -192,13 +192,16 @@ async def test_https_proxy_unsupported_tls_in_tls(
...
+    with pytest.warns(
+        RuntimeWarning,
+        match=expected_warning_text,
+    ), pytest.raises(
         ClientConnectionError,
         match=expected_exception_reason,
     ) as conn_err:
         await sess.get(url, proxy=secure_proxy_url, ssl=client_ssl_ctx)
 
-    assert type(conn_err.value.__cause__) == TypeError
```
Comment (webknjaz) [inline tests/test_proxy_functional.py:201]: I'm pretty sure the exact class is expected here..

File: tests/test_circular_imports.py (hunk 2 above)
Comment (Dreamsorcerer) [reply tests/test_circular_imports.py:34]: Isn't this one just black formatting? I didn't touch this one.

File: aiohttp/web_response.py (hunk 1 above)
Comment (Dreamsorcerer) [reply aiohttp/web_response.py:179]: > Are integers boolean

No, other way round. bool is subclass of int, so this shoudn't make any difference. It's also ancient code that could be removed at this point anyway..

File: tests/test_proxy_functional.py (hunk 3 above)
Comment (Dreamsorcerer) [reply tests/test_proxy_functional.py:201]: Didn't think it would make a difference. Feel free to revert and add a noqa.
